### ISSUE TITLE
Cannot install rumdl with brew v6

### ISSUE THREAD

**Issue by fibo** [rvben/rumdl#668]
### rumdl version

0.1.94

### Installation method

Homebrew

### Bug description

It may be related to recent [brew v6 major update](https://brew.sh/2026/06/11/homebrew-6.0.0/) which introduces "Tap trust".

If I run the command I see on [rumdl installation page](https://rumdl.dev/getting-started/installation/)

```sh
brew install rvben/tap/rumdl
```

I get this

> Warning: No available formula or cask with the name "rvben/tap/rumdl". Did you mean rvben/tap/verg?

I am using brew v6

```sh
$ brew --version
Homebrew 6.0.1-74-gecbb698
```

For now, I run the following commands to install rumdl

```sh
brew tap rvben/rumdl      
brew trust rvben/rumdl                                                  
brew install rumdl  
```

but this will "trust the whole tap".

You may want to update the documentation with something else.

### Steps to reproduce

On a Macos launch the Homebrew command documented here: https://rumdl.dev/getting-started/installation/

Use `brew` version 6.

```sh
brew install rvben/tap/rumdl
```
 

### Expected behavior

brew installs rumdl formula as usual

### Actual behavior

brew v6 does not install rumdl and ends with this message

> Warning: No available formula or cask with the name "rvben/tap/rumdl". Did you mean rvben/tap/verg?

### Configuration

```toml
config does not apply to this issue
```

### Operating system

macOS

**Comment by rvben** [comment]
Thanks for the report, @fibo! This is a docs bug, not a Homebrew 6 tap-trust issue.

The docs pointed at the wrong tap. rumdl is now in **homebrew-core**, so just use:

```sh
brew install rumdl
```

homebrew-core is trusted by default, so there's no tap-trust step under brew 6, and it ships prebuilt bottles for common platforms. I've corrected the docs. Thanks for flagging it!

**Comment by fibo** [comment]
@rvben thanks for the reply, I needed it cause I have a script that installs `rumdl` so even better that I can just use `brew install rumdl`.
